docs/scos/dev/front-end-development/202307.0/oryx/building-components/component-types.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Oryx Component types"
+description: Oryx components compared to Atomic Design
+last_updated: Sept 23, 2023
+template: concept-topic-template
+---
+
+Oryx applications are 100% built out of components. Whether it's a page or a section of the page or a button, they're all components.
+
+Oryx supports three types of components:
```

Comment:
Do you want this to be styled as a glossary?

Reply:
I'm not familiar with that,  can you link an example?